### ISSUE TITLE
Changing styles at runtime

### ISSUE THREAD

**Issue by bradmartin** [NativeScript/NativeScript#7106]
**Is your feature request related to a problem? Please describe.**
No

**Describe the solution you'd like**
The groundwork to change styles at runtime has been done and I believe exists in the core-modules already. Since we can livesync/hmr styles during development. So just thinking that the functionality to do this could be exposed on the `application` module so it can be used during `application` events in the `app.ts`. Currently there is the `changeCssFile` method but it's on `View` class. Which doesn't allow for a 'theme' approach globally at runtime. I'm currently using the `nativescript-themes` plugin which works fine but obviously if the functionality is already in the core-modules to handle this, not having to bring in another dependency would be great for the framework.

So to summarize: I'm proposing that the `changeCssFile` be exposed at the app level for theming the entire app. The process would be:

```typescript
application.setCssFileName('some-theme.css');
application.changeCssFile();
```

**Additional context**
@vchimev and I had brief chat about this on slack the other day so he can provide more context.

**Comment by NathanaelA** [comment]
@bradmartin - Just a fyi; I do several things in my themes plugin that aren't in the core framework.   So if you actually wanted this, as seemless as it is -- in the framework we would have to add some additional code to the framework.    I would be very willing to help move this plugin into the framework.

**Comment by bradmartin** [comment]
@NathanaelA - ah good to know. Was just thinking from the methods already in the core, I expected those to work similar to what your plugin was doing. So one less thing to maintain via "community" if it's already 90% in the core and working.

**Comment by rigor789** [comment]
This is supported by plugins, for example: https://github.com/NativeScript/plugins/blob/main/packages/theme-switcher/README.md
